lib/recommendation-server.js

```diff
@@ -82,17 +89,36 @@ RecommendationServer.prototype = {
       req.send();
     }
   },
+
+  getResultClass: function(data) {
+    let types = Object.keys(data.enhancements);
+    if (types.indexOf('tld') > -1) {
+      return this.resultClasses.tld;
+    } else if (types.indexOf('wikipedia') > -1) {
+      return this.resultClasses.wikipedia;
+    }
+    return null;
```

Comment:
Per my other comment, I actually think this should be moved to the recommendation.js class, since that class has the responsibility of picking a UI class that fits the data. The recommendation-server should just abstract out the service endpoint and return nicely formatted data (IMO)